npm-shrinkwrap.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "fxa-content-server",
-  "version": "1.130.1",
```

Comment:
Can you revert any changes to the npm-shrinkwrap.json file, this file should only be updated whenever new dependencies are installed or dependencies are updated. `npm install` will cause this to be updated automatically whereas `npm ci` will not.